### ISSUE TITLE
runtime build error

### ISSUE THREAD

**Issue by otrack** [lithops-cloud/lithops#432]
The command "lithops runtime build" fails with a TypeError (build_runtime() takes 1 positional argument but 3 were given) whatever follows it. An example is below. 
[log.txt](https://github.com/lithops-cloud/lithops/files/5507238/log.txt)

**Comment by gilv** [comment]
@otrack i usually navigate into runtime/<backend> and then use

     lithops runtime  build <docker space>/<image name>:<version number> -f <path to Docker file>

**Comment by aitorarjona** [comment]
@otrack GCP Functions backend does not support runtimes based on docker.
I'm now working on a patch to permit building runtimes with extra modules defined in a `requirements.txt` file instead of a Dockerfile.

**Comment by aitorarjona** [comment]
@otrack Implemented custom runtime based on a `requirements.txt` file in #434.
Please follow instructions [here](https://github.com/lithops-cloud/lithops/tree/master/runtime/gcp_functions) to build a custom runtime for Google Cloud Functions.

**Comment by otrack** [comment]
It does not seem to work (see log below). I am running lithops from the sources.

$> lithops runtime build -f requirements.txt my_matplotlib_runtime
2020-11-14 17:47:34,901 [INFO] lithops.config -- Lithops v2.2.6.dev0
2020-11-14 17:47:34,901 [DEBUG] lithops.config -- Loading configuration
2020-11-14 17:47:34,901 [INFO] lithops.config -- Getting configuration from /home/otrack/.lithops/config
2020-11-14 17:47:34,915 [DEBUG] lithops.config -- Loading Serverless backend module: gcp_functions
2020-11-14 17:47:35,405 [DEBUG] lithops.config -- Loading Storage backend module: gcp_storage
2020-11-14 17:47:35,418 [INFO] lithops.serverless.backends.gcp_functions.gcp_functions -- lithops v2.2.6.dev0 init for GCP Functions - Project: pro-gecko-137023 - Region: europe-west2

**Comment by aitorarjona** [comment]
@otrack Sorry for the late response. There was a lack of logging in GCP Functions Backend, so although the runtime was built successfully, there were no logs that reflect it. This has been fixed in #452. Could you try again? try running
```
$ lithops runtime build -f requirements.txt my_matplotlib_runtime
```
and then
```
$ lithops runtime create my_matplotlib_runtime
```
Thanks!
